Document/0x05b-Basic-Security_Testing.md

```diff
@@ -115,13 +115,36 @@ Nevertheless several hardware characteristics can be emulated, like GPS<sup>[6]<
 -- TODO : Cf testing methodologies from CEH, ... : map attack surface (Local and Remote) through Passive and Active Reconnaissance, Scanning, Gaining Access, Maintaining Access, Covering Tracks. As this is generic and common to iOS, may be part of the parent chapter --
 
 ### Static Analysis
--- TODO : Description, when it comes compared to dynamic analysis and why, what it can bring --
+Static analysis is the act of looking into App components, source code and other resources without actually executing it. This test is focused on finding misconfigured or unprotected Android IPC components as well as finding programming mistakes such as misuse of cryptography routines, find libraries with known vulnerabilities and even dynamic code loading routines.
+
+When we are performing static analysis of applications, several tools can be used. The most commonly used are Static Analyzers. The most useful ones are Androbugs, Linkedin's QARK, Android Lint and the paid HP's Fortify. However there are a plethora of static code analyzers that can be used.
```

Comment:
I would avoid saying "most useful", and we don't want to promote any specific tools, especially paid ones like Fortify.